build.gradle

```diff
@@ -4,7 +4,25 @@ buildscript {
 }
 
 plugins {
-    id 'org.jenkins-ci.jpi' version '0.38.0'
+    // this version (0.43.0 and up) , uses a later version of test-harness eliminating the 
+    // dependency on a compromised version of apache commons-text.  Shows as a red-herring in scans
+    id 'org.jenkins-ci.jpi' version '0.46.0'
+}
+
+java {
+    registerFeature('workflowJob') {
+        usingSourceSet(sourceSets.main)
+    }
+    registerFeature('workflowJobDsl') {
+        usingSourceSet(sourceSets.main)
+    }
+    registerFeature('workflowCps') {
+        usingSourceSet(sourceSets.main)
+    }
+    registerFeature('workflowStepApi') {
+        usingSourceSet(sourceSets.main)
+    }
+    
 }
```

Comment:
I assume the addition of the registerFeature included above came from your conversation with the person who works on the jpi plugin. Did this person have any website or documentation were the above was noted. If so, can you include a link to that in the Jira ticket this change is associated with? If not, :shrug:, nothing we can do about that.

Reply:
Hi Dana.  That is correct.  In fact he mentioned that he had updated the migration guide for this.